src/pages/send/confirm/confirm.html

```diff
@@ -123,20 +127,18 @@
       </div>
     </button>
 
-    <button ion-item (click)="showDescriptionPopup(tx)">
-      <span *ngIf="!tx.description" translate>Add Memo</span>
-      <span *ngIf="tx.description" translate>Memo</span>
-      <div class="item-memo">
-        <ion-note>{{tx.description}}</ion-note>
-      </div>
-    </button>
+    <editable-item (focus)="memoFocused = $event" (valChange)="tx.description = $event" [value]="tx.description" [itemPlaceholder]="'Enter a transaction memo'">
```

Comment:
item placeholder should be {{'Enter a transaction memo' | translate}} -> use npm run extract to test that the string is added to the pot file